### ISSUE TITLE
Location timestamp

### ISSUE THREAD

**Issue by RyanTG** [scottwainstock/pbm#385]
A location timestamp that is updated when: a machine is added or removed; a machine comment is made (could also include when a high score or picture or location description is added).

"Location updated mm/dd/yyyy"

We can put it underneath "Click to enter location description/hours/etc"

**Comment by RyanTG** [comment]
Related suggestion: include a "confirm" button that, when clicked, updates the timestamp. I think it could be useful (and it seems easy to add).

Underneath "Location updated mm/dd/yyyy"
"Click to confirm that this location's machine list is up to date"

(that is a bit wordy... I'll think of an edit)

**Comment by scottwainstock** [comment]
I think I have all the logic in : https://github.com/scottwainstock/pbm/tree/issue_385

... but, would you mind pulling that and putting some styling on it? The "push to confirm" and "last date updated" stuff looks pretty bad.

**Comment by RyanTG** [comment]
Yes, will do!

**Comment by RyanTG** [comment]
I'll try to commit my changes today. I pulled it and looked at a location this morning, but I didn't see "location updated mm/dd/yyyy." Does that only show up after you've clicked "confirm" once?

**Comment by scottwainstock** [comment]
Yeah it doesn't update with ajax... so if you click "confirm", you won't see the date until after the page reloads. I can fix that later.

**Comment by RyanTG** [comment]
And we already have a "location updated" timestamp, right? Like, that data was there for each location before this issue (I see it when I go to edit a location on the admin page). Can we populate each location with that data?

**Comment by scottwainstock** [comment]
Not really. That's a rails thing, and I don't think we've plugged into it
very much. It's possible it's only ever set when the object is CREATED (I
think). It might be that it got updated when location.description was
updated, but I'm not sure.

At any rate, it definitely was never updated when machines were
added/remove from a location. So, wouldn't it be misleading if that field
SOMETIMES indicates one thing and SOMETIMES indicates another?

**Comment by RyanTG** [comment]
Oh yeah, that would be misleading. I'm in agreement. I've seen jah light.

**Comment by scottwainstock** [comment]
4175534b5bd48b95f800b4bff2196d184f6ced2f

**Comment by RyanTG** [comment]
Should we keep this open for the ajax updating? Right now when you click it, nothing appears to happen.

**Comment by scottwainstock** [comment]
I'd prefer a new ticket, plz.
